integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/addsegment/AddSegmentTestCase.scala

```diff
@@ -301,6 +300,8 @@ class AddSegmentTestCase extends QueryTest with BeforeAndAfterAll {
     checkAnswer(sql("select empname from addsegment1 where empname='arvind'"), Seq(Row("arvind"),Row("arvind")))
     checkAnswer(sql("select count(empname) from addsegment1"), Seq(Row(20)))
     checkAnswer(sql("select count(*) from addsegment1"), Seq(Row(20)))
+    val showSeg = sql("show segments for table addsegment1").collectAsList()
+    assert(showSeg.get(0).get(6).toString.equalsIgnoreCase("3.75KB"))
```

Comment:
It is better to get the size from FS instead of hard coded it, this testcase failed in my laptop.
And please add assert to index size also